droidmediacamera2.cpp

```diff
@@ -1684,10 +1710,12 @@ char *droid_media_camera_get_parameters(DroidMediaCamera *camera)
             }
             break;
         }
+#if ANDROID_MAJOR >= 9
         case ACAMERA_CONTROL_AE_COMPENSATION_RANGE:
```

Comment:
mh, this is strange, because I get an error if this is not guarded

´´´
external/droidmedia/droidmediacamera2.cpp:1814:7: error: use of undeclared identifier 'ACAMERA_CONTROL_ZOOM_RATIO_RANGE'; did you mean 'ACAMERA_CONTROL_AE_COMPENSATION_RANGE'?
        case ACAMERA_CONTROL_ZOOM_RATIO_RANGE:
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             ACAMERA_CONTROL_AE_COMPENSATION_RANGE